### PR TITLE
Update containerd and snapshotters

### PR DESCRIPTION
**Pull request by AkihiroSuda**
- containerd daemon: v1.5.8
- containerd client library: v1.6.0-beta.3
- stargz-snapshotter: v0.10.1
- fuse-overlayfs-snapshotter: v1.0.4

### CODE REVIEW

**Comment by ktock** [comment]
Released binaries of stargz snapshotter v0.10.1 https://github.com/containerd/stargz-snapshotter/releases/tag/v0.10.1

File: go.mod [hunk 1]
```diff
@@ -4,26 +4,25 @@ go 1.16
 
 require (
 	github.com/Microsoft/go-winio v0.5.1
-	github.com/compose-spec/compose-go v1.0.4
+	github.com/compose-spec/compose-go v1.0.5
 	github.com/containerd/cgroups v1.0.2
 	github.com/containerd/console v1.0.3
-	github.com/containerd/containerd v1.6.0-beta.2.0.20211112054404-aa1b0736165c
-	github.com/containerd/containerd/api v1.6.0-beta.2.0.20211112054404-aa1b0736165c
+	github.com/containerd/containerd v1.6.0-beta.2.0.20211119054103-db9c6fc72276
```
Comment (AkihiroSuda) [inline go.mod:10]: Waiting for beta.3 to be tagged https://github.com/containerd/containerd/pull/6267

**Review by ktock — APPROVED** [review]

**Comment by AkihiroSuda** [comment]
Potential regression in beta.3 ?

```
   run_restart_test.go:86: the container does not seem to be restarted
--- FAIL: TestRunRestart (181.36s)
```

**Comment by AkihiroSuda** [comment]
Skipped 1.6.0-beta.3 in CI.

**Pull request merged**